src/kili/queries/asset/media_downloader.py

```diff
@@ -30,7 +30,7 @@ def get_download_assets_function(
         return None
     projects = list(
         ProjectQuery(kili.auth.client)(
-            ProjectWhere(project_id=project_id), fields, QueryOptions(disable_tqdm=True)
+            ProjectWhere(project_id=project_id), ["inputType"], QueryOptions(disable_tqdm=True)
```

Comment:
this means that `fields` should be removed in line 44?

Reply:
no, this fields is for querying the assets (content, jsoncontent, externalid, etc)